net.go

```diff
@@ -164,19 +163,21 @@ func (l *Listener) withTLS(conn net.Conn) (*tls.Conn, error) {
 		err = tconn.HandshakeContext(ctx)
 		cancel()
 	}
-	if err != nil {
-		if l.Callbacks != nil {
-			l.Callbacks.OnTLSHandshakeError(tconn, err)
-		}
-	}
 
 	return tconn, err
 }
 
 func (l *Listener) Addr() net.Addr {
+	if l.listener == nil {
+		return nil
+	}
 	return l.listener.Addr()
 }
 
 func (l *Listener) Close() error {
+	if l.listener == nil {
+		return nil
+	}
+
 	return l.listener.Close()
```

Comment:
`Addr()` has no space between `if` and `return`  when `Close()` has. Please unify it.